src/Cake.Common/Net/DownloadFileSettings.cs

```diff
@@ -18,5 +18,10 @@ public sealed class DownloadFileSettings
         /// Gets or sets the Password to use when downloading the file
         /// </summary>
         public string Password { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value that controls whether default credentials are sent when downloading the file
```

Comment:
Since this is a boolean property, the summary text should start with `Gets or sets a value indicating whether`